src/puppetlabs/puppetdb/http/fact_names.clj

```diff
@@ -25,7 +25,8 @@
                  version
                  (http-q/validate-distinct-options! (merge (keywordize-keys params) puppetdb-query))
                  (:scf-read-db globals)
-                 (:url-prefix globals))))
+                 (:url-prefix globals)
+                 (:pretty-print globals))))
```

Comment:
Looks like it does do *something*, i.e. it adds some whitespace in between the list entries.